stylist/api/views.py

```diff
@@ -17,7 +17,7 @@ def perform_create(self, serializer):
           else:
                site = Site.objects.get_current()
           instance = serializer.save(site=site)
-          if not getattr(settings, 'STYLIST_IGNORE_SASS', False):
+          if not getattr(settings, 'STYLIST_IGNORE_SASS', True):
```

Comment:
This would be a bit cleaner with the other naming convention and with out so many "nots"